### PR TITLE
Next gen

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,4 @@ language: node_js
 node_js:
   - 'stable'
   - '4.2'
-  - '0.10'
```
Comment (acamino) [inline .travis.yml]: Cool. Thanks!

File: README.md [hunk 2]
```diff
@@ -22,37 +26,43 @@ An example application implementing an automated phone line using Twilio and Exp
    $ mongod
    ```
 
-2. Clone this repository and `cd` into its directory:
+1. Clone this repository and `cd` into its directory:
    ```
```
Comment (acamino) [inline README.md]: We'd like to make the READMEs more visually appealing by highlighting code blocks. Would you mind using **```bash**?

File: README.md [hunk 3]
```diff
@@ -22,37 +26,43 @@ An example application implementing an automated phone line using Twilio and Exp
...
    $ git clone git@github.com:TwilioDevEd/ivr-recording-node.git
    $ cd ivr-recording-node
    ```
 
-3. To seed the initial data into the database by running the following:
+1. The file `seed/agents.js` contains the agents phone numbers. Replace any of these phone numbers with yours. Then seed the initial data into the database by running the following:
    ```
```
Comment (acamino) [inline README.md]: ditto

File: routes/extension.js [hunk 4]
```diff
@@ -22,7 +22,10 @@ router.post('/connect', twilio.webhook({validate: false}), function (req, res) {
     twiml
       .say('You\'ll be connected shortly to your planet.',
            { voice: 'alice', language: 'en-GB' })
-      .dial({ action: '/agents/call?agentId=' + agent.id }, function() {
+      .dial({
+        action: '/agents/call?agentId=' + agent.id,
```
Comment (acamino) [inline routes/extension.js]: You're already editing this code block. Did you consider using template literals?

File: seed/agents.js [hunk 5]
```diff
@@ -2,8 +2,8 @@ db.dropDatabase();
 
 var agents = [
   { extension: 'Brodo',   phoneNumber: '+15552483591' },
-  { extension: 'Dugobah', phoneNumber: '+15558675309' },
-  { extension: '113',     phoneNumber: '+15553185602' }
+  { extension: 'Dagobah', phoneNumber: '+15558675309' },
```
Comment (acamino) [inline seed/agents.js]: Thanks for fixing the typo! 😉

File: seed/agents.js [hunk 6]
```diff
@@ -2,8 +2,8 @@ db.dropDatabase();
...
+  { extension: 'Oober',     phoneNumber: '+15553185602' }
```
Comment (acamino) [inline seed/agents.js]: To align or not to align, that's the question. _Keep the alignment consistent please_.

**Comment by juancarlospaco** [comment]
Looks great to me.

File: routes/extension.js (hunk 4 above)
Comment (hortega) [inline routes/extension.js]: I did many times along these tutorials :smile:, but I thought we agreed on not modifying the code to use ES6 features

**Comment by acamino** [comment]
Thanks @hortega! LGTM 😉

**Pull request merged**